packages/framework/src/Component/FlashMessage/FlashMessageTrait.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Component\FlashMessage;
+
+/**
+ * @property \Psr\Container\ContainerInterface $container
+ */
+trait FlashMessageTrait
+{
+    /**
+     * @param string $template
+     * @param array $parameters
+     */
+    public function addSuccessFlashTwig(string $template, array $parameters = []): void
+    {
+        $this->addSuccessFlash($this->renderStringTwigTemplate($template, $parameters));
+    }
+
+    /**
+     * @param string $template
+     * @param array $parameters
+     */
+    public function addErrorFlashTwig(string $template, array $parameters = []): void
+    {
+        $this->addErrorFlash($this->renderStringTwigTemplate($template, $parameters));
+    }
+
+    /**
+     * @param string $template
+     * @param array $parameters
+     */
+    public function addInfoFlashTwig(string $template, array $parameters = []): void
+    {
+        $this->addInfoFlash($this->renderStringTwigTemplate($template, $parameters));
+    }
+
+    /**
+     * @param string $message
+     */
+    public function addErrorFlash(string $message): void
+    {
+        $this->addFlashMessage(FlashMessage::KEY_ERROR, $message);
+    }
+
+    /**
+     * @param mixed $message
+     */
+    public function addInfoFlash($message): void
+    {
+        $this->addFlashMessage(FlashMessage::KEY_INFO, $message);
+    }
+
+    /**
+     * @param mixed $message
+     */
+    public function addSuccessFlash($message): void
+    {
+        $this->addFlashMessage(FlashMessage::KEY_SUCCESS, $message);
+    }
+
+    /**
+     * @param string $type
+     * @param mixed $message
```

Comment:
can it be trully mixed?

Reply:
I think so. In our custom methods (add*Flash) we want parameter as string, but in this one method we want mixed parameter, becouse `flashBag()->add` has as second parameter mixed property.

Reply:
Well I think we can strictly say we want string here because I do not know what would happen if object was sent here.

Reply:
Ok - 45dc79c5c63db2c0ec61ee68e0e4fd4b523259eb